e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -897,20 +897,20 @@
             "TestReportType": "DirectMethodConnectivityReport",
             "SenderSource": "directMethodSender1.send",
             "ReceiverSource": "directMethodReceiver1.receive",
-            "TolerancePeriod": "00:01:00",
+            "TolerancePeriod": "00:03:00",
```

Comment:
What does the tolerance period do? And why are we increasing it?

Reply:
From my understanding , When Hugues originally did testing on this, he noticed that there seemed to be some arbitrary tolerance period implemented for when the directmethod test messages started coming in. It had to be 1 minute after network controller was created. Doing some investigating, some of the messages started coming in after 1 minute and the tests were considering this an error. Increasing the tolerance period to 3 minutes would catch these